EmailSender/Api/TemplateRepositoryInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\EmailSender\Api;
+
+use Magento\Framework\Exception\NoSuchEntityException;
+
+/**
+ * @interface
+ */
+interface TemplateRepositoryInterface
```

Comment:
Overcomplicated code - overcomplicated and it actually misses the approach, which is system.xml + config.xml + reading value for email template from config, see https://drive.google.com/file/d/1ed3KeOMXJoXriWg7CJeEh4rEGzFkU1vk/view?usp=share_link

Reply:
Thanks for the valuable feedback. I'll dive into this and do recommended rectifications and modification. I tried to update the email template on every update without loading every template in separate configuration. But unfortunately i won't be able to do that. I was thinking to automate approach with lesser and optimized code